independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/QuarkusProjectHelper.java

```diff
@@ -1,36 +1,47 @@
 package io.quarkus.devtools.project;
 
+import static io.quarkus.platform.catalog.processor.CatalogProcessor.getCodestartArtifacts;
+
 import io.quarkus.bootstrap.resolver.maven.BootstrapMavenException;
 import io.quarkus.bootstrap.resolver.maven.MavenArtifactResolver;
 import io.quarkus.bootstrap.util.DependencyNodeUtils;
 import io.quarkus.devtools.messagewriter.MessageWriter;
 import io.quarkus.devtools.project.extensions.ExtensionManager;
 import io.quarkus.platform.catalog.processor.ExtensionProcessor;
 import io.quarkus.platform.descriptor.loader.json.ClassPathResourceLoader;
+import io.quarkus.platform.descriptor.loader.json.ResourceLoader;
 import io.quarkus.platform.tools.ToolsUtils;
 import io.quarkus.registry.ExtensionCatalogResolver;
 import io.quarkus.registry.RegistryResolutionException;
 import io.quarkus.registry.catalog.Extension;
 import io.quarkus.registry.catalog.ExtensionCatalog;
+import io.quarkus.registry.config.PropertiesUtil;
 import io.quarkus.registry.config.RegistriesConfig;
 import io.quarkus.registry.config.RegistriesConfigLocator;
 import java.net.URL;
 import java.net.URLClassLoader;
 import java.nio.file.Path;
-import java.util.Arrays;
+import java.util.ArrayList;
 import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
 import org.eclipse.aether.artifact.Artifact;
 
 public class QuarkusProjectHelper {
 
-    private static final String CODESTARTS_ARTIFACTS = "codestarts-artifacts";
-
+    private static final String BASE_CODESTART_ARTIFACT_COORDS_PROPERTY = "BASE_CODESTART_ARTIFACT_COORDS";
```

Comment:
Properties usually have lower-case names.

Reply:
I changed the way it works, using the properties was simpler and easier to override.